### PR TITLE
Added calico limits and update to 3.2.3 bugfix release

### PR DESCRIPTION
**Pull request by r7vme**
Part of: https://github.com/giantswarm/giantswarm/issues/4159

### CODE REVIEW

File: service/controller/v4/version_bundle.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func VersionBundle() versionbundle.Bundle {
 		Components: []versionbundle.Component{
 			{
 				Name:    "calico",
-				Version: "3.0.8",
+				Version: "3.2.0",
```
Comment (r7vme) [inline service/controller/v4/version_bundle.go]: Align with reality

File: service/controller/v5/cloudconfig/template.go [hunk 2]
```diff
@@ -195,6 +196,8 @@ spec:
         # if it ever gets evicted.
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
+      nodeSelector:
```
Comment (r7vme) [inline service/controller/v5/cloudconfig/template.go]: From 3.2.3 manifest, just to be aligned

File: service/controller/v4/version_bundle.go (hunk 1 above)
Comment (pipo02mix) [reply service/controller/v4/version_bundle.go]: 3.2.3?

Comment (r7vme) [reply service/controller/v4/version_bundle.go]: 2.0.0 bundle has 3.2.0 calico, but we forgot to update this.

**Pull request merged**